src/components/fields/location-field/location-modal/location-search/location-search.styles.ts

```diff
@@ -129,10 +133,10 @@ export const ResultItem = styled.div<{ active?: boolean }>`
 	align-items: center;
 	gap: 1rem;
 	padding: 1rem 1rem 1rem 0;
-	border-bottom: 1px solid ${V2_Color.Neutral[5]};
+	border-bottom: 1px solid ${Colour.border};
 	text-transform: uppercase;
 	cursor: pointer;
-	background-color: ${({ active }) => (active ? V2_Color.Accent.Light[5] : `transparent`)};
+	background-color: ${({ active }) => (active ? Colour["bg-selected"] : `transparent`)};
 
 	.keyword {
 		font-family: "Open Sans Semibold";
```

Comment:
remove this usage of font-family and use the `Font.Spec["weight-semibold"]` token instead to set the font weight